src/form.jsx

```diff
@@ -216,6 +221,20 @@ class ReactForm extends React.Component {
     }
   }
 
+  handleBlur(event) {
+    // Call submit function on blur
+     const { onBlur } = this.props;
+        const data = this._collectFormData(this.props.data);
+        onBlur(data);
```

Comment:
This will broke if onBlur function is not supplied. So you need to check existence of function like this
```js
  handleBlur(event) {
    // Call submit function on blur
    const { onBlur } = this.props;
    if (onBlur) {
      const data = this._collectFormData(this.props.data);
      onBlur(data);
    }
  }
```